src/main/java/by/exadel/internship/entity/Internship.java

```diff
@@ -0,0 +1,55 @@
+package by.exadel.internship.entity;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.time.LocalDate;
+import java.util.UUID;
+
+@Entity
+@Table(name = "internship")
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Internship {
+
+    @Id
+    @Column(name = "inship_id")
+    private UUID id;
```

Comment:
provide lines between fields for readability

Reply:
done
![image](https://user-images.githubusercontent.com/65280608/113557714-24b96780-9607-11eb-96b6-42b59ca72cc3.png)